src/Api/Model/Shared/UserListModel.php

```diff
@@ -10,7 +10,7 @@ public function __construct()
     {
         parent::__construct(
             UserModelMongoMapper::instance(),
-            ["username" => ['$regex' => ""], "isDeleted" => false],
+            ["username" => ['$regex' => ""], "isDeleted" => ["$in" => [false, null]]],
```

Comment:
This looks correct, however a failing unit test reveals that this likely needs to be in single quotes, otherwise $in is interpreted as a PHP variable instead of a mongo operator.
```suggestion
            ["username" => ['$regex' => ""], "isDeleted" => ['$in' => [false, null]]],
```

Reply:
Unit test failure:
```
There was 1 error:

1) MultipleModelTest::testUserList_HadOnlyUsers
Undefined variable: in

/var/www/html/Api/Model/Shared/UserListModel.php:13
/var/www/test/php/model/shared/MultipleModelTest.php:38
phpvfscomposer:///var/www/html/vendor/phpunit/phpunit/phpunit:97
```